osu.Game/Screens/OnlinePlay/DrawableRoomPlaylistItem.cs

```diff
@@ -165,18 +170,37 @@ private void refresh()
                                 {
                                     AutoSizeAxes = Axes.Both,
                                     Direction = FillDirection.Horizontal,
-                                    Spacing = new Vector2(15, 0),
+                                    Spacing = new Vector2(10f, 0),
                                     Children = new Drawable[]
                                     {
-                                        authorText = new LinkFlowContainer { AutoSizeAxes = Axes.Both },
-                                        modDisplay = new ModDisplay
+                                        new FillFlowContainer
+                                        {
+                                            AutoSizeAxes = Axes.Both,
+                                            Direction = FillDirection.Horizontal,
+                                            Spacing = new Vector2(10f, 0),
+                                            Children = new Drawable[]
+                                            {
+                                                authorText = new LinkFlowContainer { AutoSizeAxes = Axes.Both },
+                                                explicitPill = new ExplicitBeatmapPill
+                                                {
+                                                    Alpha = 0f,
+                                                    Anchor = Anchor.CentreLeft,
+                                                    Origin = Anchor.CentreLeft,
+                                                    Margin = new MarginPadding { Top = 3f },
+                                                }
```

Comment:
Not sure I agree with the placement on this one. Both on beatmap set overlay and on beatmap panel, the explicit label is next to the author/title of the beatmap track, while here it's next to the mapper.

Reply:
That's intentional given that the above isn't just a title, it is `Title - Artist [Difficulty]`, which fills up the screen in almost all cases.